### DISCUSSION TITLE
How do I disable "system.runner.lowdiskspacethreshold"?

### DISCUSSION THREAD

**Discussion by Hubro** [community/community#160449] [Actions]
I'm using Firecracker VMs with read-only root drives as ephemeral GitHub runners to execute workflows. This works well, but I'm flooded with warnings about running out of disk space, because the read-only root drive has 0MB free space.

Digging through the source code I can see that low disk space notifications can be disabled by changing/removing the "system.runner.lowdiskspacethreshold" variable, but I don't know what a "variable" is or how to modify it.

Is it something I can do with configuration or do I have to apply a patch?

**Comment by Peter-McLean-Altera** [comment]
We are experiencing this same problem
